charts/cp-schema-registry/values.yaml

```diff
@@ -72,8 +89,8 @@ affinity: {}
 ## Privilege and access control settings for a Pod or Container
 ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 securityContext:
-  runAsUser: 10001
-  runAsGroup: 10001
+  runAsUser: 1000
```

Comment:
before this change, schema-registry was trying to write files to disk and failing on permissions. Blindly supplied this 1000 values from a stackoverflow post, needs validating please